lib/deliver/app_screenshot.rb

```diff
@@ -113,6 +117,11 @@ def self.calculate_screen_size(path)
           [1536, 2008],
           [1536, 2048]
         ],
+        ScreenSize::IOS_IPAD_PRO => [
+          [2732, 2048],
+          [2048, 2732]
+          # missing resolutions with status bar, not sure what they should be
```

Comment:
not sure I understand. I was able to upload raw size screenshots (2732x2048). The other sizes contain both raw sizes + sizes reduced because the status bar is showing. I am not sure how big in pixels the status bar will be in these devices to be able to add these resolutiosn. Maybe one can test in the emulator, but I am still in Xcode 6.4 here. I googled around and couldn't find the information. We could ship without those resolutions and add them when we know them.